components/Home/sidebar/RootLeftSidebar.tsx

```diff
@@ -0,0 +1,436 @@
+import { breakpoints } from "~/config/themes/screen";
+import { css, StyleSheet } from "aphrodite";
+import { NAVBAR_HEIGHT } from "~/components/Navbar";
+import { AnimatePresence, motion } from "framer-motion";
+import {
+  getCurrMediaWidth,
+  useEffectOnScreenResize,
+} from "~/config/utils/useEffectOnScreenResize";
+import { filterNull, isEmpty } from "~/config/utils/nullchecks";
+import { getCurrentUser } from "~/config/utils/getCurrentUser";
+import { NextRouter, useRouter } from "next/router";
+import {
+  ReactElement,
+  SyntheticEvent,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+import ALink from "~/components/ALink";
+import colors from "~/config/themes/colors";
+import icons from "~/config/themes/icons";
+import RHLogo from "~/components/Home/RHLogo";
+import RootLeftSidebarItem, {
+  ITEM_FADE_DURATION,
+  Props as RootLeftSidebarItemProps,
+} from "./sidebar_items/RootLeftSidebarItem";
+
+type Props = {};
+
+const getLeftSidebarItemAttrs = ({
+  currentUser,
+  isMinimized,
+  router,
+}: {
+  currentUser: any;
+  isMinimized: boolean;
+  router: NextRouter;
+}): RootLeftSidebarItemProps[] => {
+  const { pathname = "" } = router ?? {};
+  const { organization_slug = "", id } = currentUser ?? {};
+  const isLoggedIn = !isEmpty(id);
+
+  return filterNull([
+    {
+      icon: icons.home,
+      label: "Home",
+      isActive: ["", "/"].includes(pathname),
+      isMinimized,
+      onClick: (event: SyntheticEvent): void => {
+        event.preventDefault();
+        router.push("/");
+      },
+    },
+    {
+      icon: icons.squares,
+      label: "Hubs",
+      isActive: ["/hubs"].includes(pathname),
+      isMinimized,
+      onClick: (event: SyntheticEvent): void => {
+        event.preventDefault();
+        router.push("/hubs");
+      },
+    },
+    isLoggedIn
```

Comment:
We should be showing the notebook even if the user is not logged in. See what we do currently in the top nav "Menu" link and duplicate that logic. Reason being is that we want to signal to users that we have that feature